src/builder/commands/copy.rs

```diff
@@ -85,6 +91,7 @@ impl Copy {
         .member("include_regex", V::Scalar::new().optional())
         .member("owner_uid", V::Numeric::new().min(0).optional())
         .member("owner_gid", V::Numeric::new().min(0).optional())
+        .member("umask", V::Numeric::new().min(0).max(0o777).optional())
```

Comment:
At first I made default mode for copy command but then decided to keep backwards compatibility. Also I didn't like idea to add yet another option.